configs/_base_/datasets/waymoD5-mv3d-3class.py

```diff
@@ -29,8 +29,9 @@
         type='RandomResize3D',
         scale=(1248, 832),
         ratio_range=(0.95, 1.05),
+        # ratio_range=(1., 1.),
```

Comment:
It seems a line of unused new code was added. Add comments to explain why if we want to keep them here.